pkgs/servers/mail/dovecot/plugins/fts_xapian/default.nix

```diff
@@ -0,0 +1,33 @@
+{ stdenv, fetchFromGitHub, autoconf, automake, pkg-config, dovecot, libtool, xapian, icu64 }:
+stdenv.mkDerivation {
+  pname = "fts-xapian";
+  version = "1.2.11a";
```

Comment:
Maybe update to https://github.com/grosjo/fts-xapian/releases/tag/1.3.

Reply:
1.3.1 got out today, even.